src/Search.php

```diff
@@ -4464,7 +4464,7 @@ public static function addDefaultWhere($itemtype)
                 break;
 
             case 'Config':
-                $availableContexts = ['core'] + Plugin::getPlugins();
+                $availableContexts = array_merge(['core', 'inventory'], Plugin::getPlugins());
```

Comment:
Legend, tested and works as intended. 

I can confirm this will resolve my bug report

Reply:
:+1: